google/cloud/storage/tests/testbench.py

```diff
@@ -173,6 +173,20 @@ def delete_acl(self, entity):
         indexed.pop(entity)
         self.metadata['acl'] = indexed.values()
 
+    def get_acl(self, entity):
+        """
+        Get a single AccessControl entry from the Object revision.
+
+        :param entity:str the name of the entity.
+        :return:dict with the contents of the ObjectAccessControl.
+        """
+        entity = self.canonical_entity_name(entity)
+        for acl in self.metadata.get('acl', []):
+            if acl.get('entity', '') == entity:
```

Comment:
Probably worth doing a case-insensitive comparison here -- calling `lower()` on both of these should be fine.

Reply:
Good idea.  What do you think of calling `lower()` in `canonical_entity_name()` instead? That way all the acl functions work like this.

Reply:
Sounds good to me (for `entity`).  I realize the `self.metadata` is populated by us here, but I'd still opt for calling `lower()` on whatever we get back from `acl.get('entity', '')` as well, in case someone adds AnEntityLikeThis@foo.com to the ACLs at some point and wonders why things are failing :)

For context, I believe the service sometimes returns entities in mixed case (I've only seen this for older entities that aren't stored in all lower case), which has resulted in similar string comparison issues in gsutil.

Reply:
Done.